torch/fx/experimental/symbolic_shapes.py

```diff
@@ -2513,6 +2513,11 @@ def hint(s):
                         track_symint(TensorPropertySource(src, TensorProperty.STRIDE, i), ss)
                     track_symint(TensorPropertySource(src, TensorProperty.STORAGE_OFFSET), curr_t.storage_offset())
 
+        # fill in any missing sources
+        for var, sources in self.var_to_sources.items():
+            if var not in symbol_to_source:
+                symbol_to_source[var].extend(sources)
+
```

Comment:
Discussing offline, this is not sound. You can't just do this because downstream bounds guards use symbol_to_source as a source of truth.

Reply:
@voznesenskym I removed this.  I think the the speculate/restart stuff it is no longer needed.